onnxruntime/test/providers/provider_test_utils.h

```diff
@@ -26,6 +26,11 @@ namespace onnxruntime {
 class InferenceSession;
 
 namespace test {
+template <typename T>
+struct SeqTensors {
+  std::vector<std::pair<std::vector<int64_t>, std::vector<T>>> tensors;
+};
```

Comment:
A comment explaing the first item in the pair is the shape would be helpful

Reply:
done